front-end/src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
+// import logo from "./../assets/images/logo.svg";
+import "./App.css";
```

Comment:
Seems like things are getting compiled to scss. Why this?

Reply:
scss files get compiled to css , which is how it is referenced in the react components.